frontend/src/api/user.js

```diff
@@ -1,7 +1,8 @@
-const USER_URL = 'http://localhost:8000/api/v1/users';
+const CREATE_USER_URL = 'http://localhost:8000/api/v1/users';
+const USER_URL = 'http://localhost:8000/api/v1/users/username';
```

Comment:
There's repeating code in these constants. Find a way these constants can re-use the same values.  Learn about DRY principle. 

https://www.baeldung.com/cs/dry-software-design-principle